apps/greencheck/tests/test_admin.py

```diff
@@ -81,9 +77,12 @@ def test_get_queryset_staff(
         assert gip in qs
 
     def test_get_queryset_non_staff(
-        self, db, rf, hosting_provider_with_sample_user, green_ip,
+        self,
+        db,
```

Comment:
If we're going to use the `django_db` mark in pytest passinf in `db` as a fixture, would you mind making the change here too so it's consistent in this PR?

Reply:
It looks like we use both version across our test cases. In each new test I went with the variant that is already used in other tests in the same module. Do you want to take the decision now about this and update all occurrences? I don't see it as important, but let me know if you think otherwise

Reply:
Nah, not's not a priority.

Let's keep this as is and create an issue to review this where make a recorded decision and then apply it to all future PRs instead. This PR is already ginormous.